### PR TITLE
Notes as events

### PR DESCRIPTION
**Pull request by ericnewcomer**

### CODE REVIEW

File: src/contacts/events.ts [hunk 1]
```diff
@@ -585,8 +576,14 @@ export const getEventGroupType = (event: ContactEvent, ticket: string) => {
     return 'messages';
   }
   switch (event.type) {
-    case Events.NOTE_CREATED:
-      return 'notes';
+    case Events.TICKET_EVENT:
```
Comment (rowanseymour) [inline src/contacts/events.ts:579]: maybe these should still be split into separate event types? The engine already produces `ticket_opened` events, so I was imagining we'd take TicketEvent instances from the DB and generate things that look like that, e.g. types like `ticket_opened`, `ticket_assigned`, `ticket_note_added`, `ticket_closed`, `ticket_reopened`

Comment (ericnewcomer) [reply src/contacts/events.ts:579]: Hmmmm.. well, geez. ok then.

Comment (rowanseymour) [reply src/contacts/events.ts:579]: Currently we're pulling https://app.rapidpro.io/mr/docs/sessions.html#event:ticket_opened from session blobs so we'd probably want something like but with new fields like `created_by`

Comment (ericnewcomer) [reply src/contacts/events.ts:579]: Right, ya, making that change. More just poking at myself for churning on this.

**Pull request merged**